### ISSUE TITLE
Training my own classifier, what is the expected loss should be?

### ISSUE THREAD

**Issue by tigerhaolu727** [davidsandberg/facenet#199]
Hi David,

I have created my own MS-1m dataset, just trying to train my own model using this dataset and following [Classifier training of inception resnet v1](https://github.com/davidsandberg/facenet/wiki/Classifier-training-of-inception-resnet-v1) after 147 epoches, the loss stay around 4.1 for a while now,  just wondering what the expected value for the loss, as my understanding the loss should be 0.0XXX. If the loss stay like 4.1 and regLoss is 3.2, what could be the problem? 

Thanks a lot if you can give any hint

**Comment by davidsandberg** [comment]
Hi,
This is the output after training for 250 000 time steps on a subset of the MS-Celeb-1M dataset. The loss is note terribly low but the model performs ok on LFW (see below).
Good luck!

```
2017-02-17 19:36:02.287+0100 [info]   [stdout] Epoch: [250][997/1000]	Time 0.392	Loss 3.418	RegLoss 2.291
2017-02-17 19:36:02.287+0100 [info]   [stdout] Epoch: [250][998/1000]	Time 0.490	Loss 3.269	RegLoss 2.291
2017-02-17 19:36:02.287+0100 [info]   [stdout] Epoch: [250][999/1000]	Time 0.522	Loss 3.221	RegLoss 2.291
2017-02-17 19:36:02.287+0100 [info]   [stdout] Epoch: [250][1000/1000]	Time 0.575	Loss 3.422	RegLoss 2.291
2017-02-17 19:36:02.287+0100 [info]   [stdout] Saving variables
2017-02-17 19:36:02.287+0100 [info]   [stdout] Variables saved in 0.59 seconds
2017-02-17 19:36:02.287+0100 [info]   [stdout] Runnning forward pass on LFW images
2017-02-17 19:36:02.287+0100 [info]   [stdout] Accuracy: 0.993+-0.004
2017-02-17 19:36:02.287+0100 [info]   [stdout] Validation rate: 0.98100+-0.01334 @ FAR=0.00100
```

**Comment by eisneim** [comment]
hi @davidsandberg, what could went wrong if my Regloss is always 0.0?
this one might already overfit(train_softmax.py)
Epoch: [164][649/1000]	Time 1.520	Loss 0.024	RegLoss 0.000
Epoch: [164][650/1000]	Time 1.074	Loss 0.024	RegLoss 0.000
Epoch: [164][651/1000]	Time 1.044	Loss 0.017	RegLoss 0.000

so I restart the training from scratch but still, the regloss is 0
Epoch: [0][133/1000]	Time 1.408	Loss 8.843	RegLoss 0.000
Epoch: [0][134/1000]	Time 1.451	Loss 9.008	RegLoss 0.000

thanks!
